duckduckgo/macos-browser
# Use GRDB 2.1.0 packaged as xcframework

Task/Issue URL: https://app.asana.com/0/414235014887631/1204022201522438/f

**Description**:
Use GRDB as binary framework to avoid recompiling sqlite3.c every time the app is built.

**Steps to test this PR**:
Smoke test everything that uses plain sqlite database via GRDB:

* Secure Vault
* Logins/Bookmarks import
* WebsiteDataStore

Note that this has already been tested in https://github.com/duckduckgo/macos-browser/pull/989 (this PR is equivalent to the old on, but has been reverted in the meantime due to issues with iOS release builds).

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
